.circleci/config.yml

```diff
@@ -53,13 +53,13 @@ jobs:
           path: ./reports
   test:
     docker:
-      - image: circleci/node:10.15.3
+      - image: circleci/node:12.16.3
         environment:
           MOCHA_OPTS: --reporter mocha-junit-reporter --reporter-options mochaFile=./reports/junit/mocha/mocha.xml
       - image: singularities/pubsub-emulator
         environment:
           PUBSUB_PROJECT_ID: pubcap-test
-          PUBSUB_LISTEN_ADDRESS: 0.0.0.0:8085
+          PUBSUB_LISTEN_ADDRESS: 0.0.0.0:8802
```

Comment:
Just curious what was the reason for changing the listen address?

Reply:
So that it matches what we use in `6mon`, so you can run `npm test` on localhost :)

Reply:
Oh, rad! That will be super useful. 💪